[2022-01-26 04:51:43,253] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-26T04:50:45.201776+00:00 [queued]>
[2022-01-26 04:51:43,284] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2022-01-26T04:50:45.201776+00:00 [queued]>
[2022-01-26 04:51:43,285] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 04:51:43,285] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-26 04:51:43,286] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-26 04:51:43,302] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): apply_model> on 2022-01-26T04:50:45.201776+00:00
[2022-01-26 04:51:43,311] {standard_task_runner.py:52} INFO - Started process 803 to run task
[2022-01-26 04:51:43,318] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'apply_model', '2022-01-26T04:50:45.201776+00:00', '--job-id', '210', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmp5q4_iu3d', '--error-file', '/tmp/tmpmpvylowi']
[2022-01-26 04:51:43,325] {standard_task_runner.py:77} INFO - Job 210: Subtask apply_model
[2022-01-26 04:51:43,383] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.apply_model 2022-01-26T04:50:45.201776+00:00 [running]> on host 115f9a25dda0
[2022-01-26 04:51:43,434] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=apply_model
AIRFLOW_CTX_EXECUTION_DATE=2022-01-26T04:50:45.201776+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-26T04:50:45.201776+00:00
[2022-01-26 04:52:22,869] {python.py:118} INFO - Done. Returned value was: None
[2022-01-26 04:52:22,901] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=apply_model, execution_date=20220126T045045, start_date=20220126T045143, end_date=20220126T045222
[2022-01-26 04:52:22,999] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-26 04:52:23,056] {local_task_job.py:146} INFO - Task exited with return code 0
